index.html

```diff
@@ -18,6 +18,11 @@
       <nav class="nav">
         <a href="#"><h1 class="my-logo">My logo</h1></a>
         <img src="assets/Unionnavbar.png" class="navbar">
+        <ul class="nav-list">
+          <li class="nav-item1">Portfolio</li>
+          <li class="nav-item2">About</li>
+          <li class="nav-item3">Contact</li>
+        </ul>
       </nav>
```

Comment:
- [ ] You should stick to the design as much as possible. Please remove the underline from your logo at the left and I would suggest please you increase the font weight of the heading and also increase the size of the `LET'S CONNECT` subheading to match the design.
- [ ] Also manage the equal gap between the social media icons.
### Your design
![his](https://user-images.githubusercontent.com/81727988/213866303-0c8717cd-10f7-48c1-ac75-e2fe1ff1ad4e.jpg)


### Figma design
![fimga](https://user-images.githubusercontent.com/81727988/213866247-570ddc30-0377-477b-958f-8b59f63ffc42.jpg)